recipes/picobench/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+from conans import CMake, ConanFile, tools
+import functools
+import os
+
+required_conan_version = ">=1.33.0"
+
+class PicobenchConan(ConanFile):
+    name = "picobench"
+    description = "A micro microbenchmarking library for C++11 in a single header file"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/iboB/picobench"
+    topics = ("benchmark", "header-only")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "with_cli": [True, False],
+        "bind_to_one_core": [True, False],
+    }
+    default_options = {
+        "with_cli": False,
+        "bind_to_one_core": False,
+    }
+    generators = "cmake"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def export_sources(self):
+        self.copy("CMakeLists.txt")
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            self.copy(patch["patch_file"])
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 11)
+
+    def package_id(self):
+        if not self.options.with_cli:
+            self.info.header_only()
```

Comment:
```suggestion
            self.info.header_only()
        else:
            del self.info.settings.compiler
```
https://github.com/conan-io/conan-center-index/blob/master/docs/packaging_policy.md#settings

The docs is wrong, `del self.settings.compiler` only works under `configure` method.

Reply:
Fixed.

I didn't read this document. I will read this. 
Thank you.